### ISSUE TITLE
Division data error from Hansard

### ISSUE THREAD

**Issue by mackaymackay** [openaustralia/publicwhip#1218]
**Context:**

Kingston MP Amanda Rishworth is listed incorrectly as crossing the floor in the [5:03pm 27 Nov 2019 division](https://theyvoteforyou.org.au/divisions/representatives/2019-11-27/15). This is not our error, but comes from [Hansard itself](https://parlinfo.aph.gov.au/parlInfo/genpdf/chamber/hansardr/a7ce6cc1-21ba-4cac-bf80-f84a4f8caf08/0218/hansard_frag.pdf;fileType=application%2Fpdf) and I've contacted the APH about it, so hopefully they'll fix it in their record. (See the [APH division page](https://www.aph.gov.au/Parliamentary_Business/Chamber_documents/HoR/Divisions/details?id=859) for confirmation that MP Rishworth did not in fact cross the floor.)

**Action:**

Are we able to manually change Rishworth's vote so that she is listed on TVFY as voting No with the rest of her party (Labor)? Or do we have to wait for APH to correct Hansard?

**Comment by jamezpolley** [comment]
> Are we able to manually change Rishworth's vote ....

Yes, we can do that with a [patch](https://github.com/openaustralia/openaustralia-parser/tree/master/data/patches); but the process to make them is fiddly.

> Or do we have to wait for APH to correct Hansard?

I had a whole thing written about how I think this already has been corrected... but then I realised I was looking at the 17:09 vote rather than 17:03. Now that I'm looking at the right vote.. no it's not corrected yet. I'll make a patch.

**Comment by jamezpolley** [comment]
I've pushed this patch live to OA, and re-parsed the speeches from that day, which has updated the generated XML. I've loaded this into TVFY, which has corrected some parts of the site but not others - for instance, https://theyvoteforyou.org.au/people/representatives/kingston/amanda_rishworth no longer lists this division in the list of rebellions, and https://theyvoteforyou.org.au/divisions/representatives/2019-11-27/15 shows that Amanda Rishworth voted No. 

However, other parts of the site rely on the cache of guessed whips, so the latter page still says "1 rebellion" even though there are no rebellions in the list, and the former page still says "rebelled 1 time" even though there are no rebellions in the list.

I've kicked off the cache update job which will fix this; it usually takes a while to run.
